### PR TITLE
ci: Add XTS Dry Run Workflow

### LINKED ISSUE #16538: Update XTS suite to enable manual dry runs on commits
Update the XTS workflow to enable dry runs. 

Adds features: 
- workflow_dispatch input: enable dry run
  - set to false by default
- workflow_dispatch input: commit_hash/ref
  - set to '' by default

### PR DESCRIPTION
**Pull request by rbarkerSL**
**Description**:

Adds a new workflow to enable test-runs of the XTS suite

**Related Issue(s)**:

Fixes #16538

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/workflows/zxf-dry-run-extended-test-suite.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+
+name: "ZXF: Extended Test Suite - Dry Run"
+on:
+  workflow_dispatch:
+    inputs:
+      commid_sha:
```
Comment (andrewb1269hg) [inline .github/workflows/zxf-dry-run-extended-test-suite.yaml:21]: Typo? `commit_sha` perhaps?

File: .github/workflows/zxf-dry-run-extended-test-suite.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    name: Execute eXtended Test Suite
+    uses: ./.github/workflows/node-zxc-compile-application-code.yaml
+    with:
+      custom-job-label: Execute eXtended Test Suite
+      enable-timing-sensitive-tests: true
+      enable-time-consuming-tests: true
+      enable-hammer-tests: true
+      enable-hapi-tests-time-consuming: true
+      enable-network-log-capture: true
+      ref: ${{ inputs.commid_sha }}
```
Comment (andrewb1269hg) [inline .github/workflows/zxf-dry-run-extended-test-suite.yaml:51]: Typo?

File: .github/workflows/zxf-dry-run-extended-test-suite.yaml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      slack-api-token: ${{ secrets.PLATFORM_SLACK_API_TOKEN }}
+      grafana-agent-username: ${{ secrets.GRAFANA_AGENT_USERNAME }}
+      grafana-agent-password: ${{ secrets.GRAFANA_AGENT_PASSWORD }}
+
+  hedera-node-jrs-panel:
+    name: Hedera Node JRS Panel
+    uses: ./.github/workflows/zxc-jrs-regression.yaml
+    with:
+      custom-job-name: "Abbrev Update Test"
+      ref: ${{ inputs.commid_sha }}
```
Comment (andrewb1269hg) [inline .github/workflows/zxf-dry-run-extended-test-suite.yaml:86]: Typo?

File: .github/workflows/zxf-dry-run-extended-test-suite.yaml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      gradle-cache-username: ${{ secrets.GRADLE_CACHE_USERNAME }}
+      gradle-cache-password: ${{ secrets.GRADLE_CACHE_PASSWORD }}
+
+  abbreviated-panel:
+    name: JRS Panel
+    uses: ./.github/workflows/zxc-jrs-regression.yaml
+    with:
+      custom-job-name: "Platform SDK"
+      panel-config: "configs/suites/GCP-PRCheck-Abbrev-4N.json"
+      ref: ${{ inputs.commid_sha }}
```
Comment (andrewb1269hg) [inline .github/workflows/zxf-dry-run-extended-test-suite.yaml:63]: Typo?

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM other than Andrew's comments. Thanks @rbarkerSL

**Pull request merged**